main/config_BT.h

```diff
@@ -48,7 +48,7 @@ extern void MQTTtoBT(char * topicOri, JsonObject& RFdata);
 #endif
 
 #ifndef pubBLEManufacturerData
-  #define pubBLEManufacturerData true // comment if you don't want to publish the manufacturer's data (in case data has characters that aren't valid with receiving client)
+  #define pubBLEManufacturerData false // define true if you want to publish the manufacturer's data (sometimes contains characters that aren't valid with receiving client)
```

Comment:
We should comment it I think, if it is only set to false it will be considered as defined in ZgatewayBT

Reply:
I couldn't find how to insert a linked code block to a different file in the reply, so I made a comment below with the justification to your comment.

If you'd still prefer commenting out the #define to disable it, I'll update ZgatewayBT,ino to match this method.